osf/utils/migrations.py

```diff
@@ -2,13 +2,32 @@
 import json
 import logging
 
+from contextlib import contextmanager
+
 from website import settings
 from osf.models import NodeLicense, MetaSchema
 from website.project.metadata.schemas import OSF_META_SCHEMAS
 
 logger = logging.getLogger(__file__)
 
 
+@contextmanager
+def disable_auto_now_fields(model):
+    """
+    Context manager to disable updates of all auto_now fields for a given model.
+
+    """
+    for field in model._meta.get_fields():
```

Comment:
I wasn't sure if this should disable all `auto_now` fields, or just specified fields. I went with disabling all fields, since that would be simplest to use, but this func could also be:
```
@contextmanager
def disable_auto_now_fields(model, fields):
    for field in model._meta.get_fields():
        if field.name in fields:
            ...
```